.travis.yml

```diff
@@ -3,8 +3,11 @@ language: java
 jdk:
   - oraclejdk8
 
-install:
-  ./gradlew assemble -i --stacktrace
+install: false
+
+script:
+  - mvn package -f maven-sdk3/pom.xml
+  - ./gradle-with-plugin/gradlew -b gradle-with-plugin/build.gradle assemble
```

Comment:
Can't you cd to the directory and then execute the command like you would normally do?

Reply:
That is indeed better